.gitignore

```diff
@@ -17,3 +17,4 @@ npm-debug.log
 *.*~
 .idea
 .oracle_jre_usage
+.vscode/
```

Comment:
Please add this to your personal gitignore rather than the project level gitignore.